bundles/org.openhab.binding.bluelink/README.md

```diff
@@ -0,0 +1,222 @@
+# Bluelink Binding (Hyundai/Genesis)
+
+This binding integrates Hyundai vehicles equipped with Bluelink connected car services.
+It allows you to monitor your vehicle's status and control various features remotely.
+
+Genesis vehicles may also work, though this has not been tested.
+
+**NOTE: The binding only support the US region at the moment, because the Bluelink API
+differs by region.**
+
+## Supported Things
+
+| Thing Type | Description                                  |
+|------------|----------------------------------------------|
+| `account`  | Bridge representing your Bluelink account    |
+| `vehicle`  | A vehicle connected to your Bluelink account |
+
+## Discovery
+
+Once an account bridge is configured and online, the binding will automatically discover vehicles
+registered to your account.
+
+## Actions
+
+Vehicle things support the following actions:
+
+* Force Refresh: Requests up-to-date data from the vehicle and updates the channels.
+* Climate Start
+* Climate Stop
+
+## Thing Configuration
+
+### Account Bridge
+
+| Parameter  | Required | Description                                              |
+|------------|----------|----------------------------------------------------------|
+| `username` | Yes      | Bluelink account email                                   |
+| `password` | Yes      | Bluelink account password                                |
+| `pin`      | No       | Bluelink service PIN (required for lock/unlock commands) |
+
+### Vehicle
+
+| Parameter              | Required | Default | Description                                       |
+|------------------------|----------|---------|---------------------------------------------------|
+| `vin`                  | Yes      | -       | Vehicle Identification Number (VIN)               |
+| `refreshInterval`      | No       | 60      | Status refresh interval in minutes (cached)       |
+| `forceRefreshInterval` | No       | 240     | Status refresh interval in minutes (from vehicle) |
+
+The `refreshInterval` parameter controls how often cached data is fetched from Kia/Hyundai servers.
+In contrast, `forceRefreshInterval` controls fetching of data from the vehicle. If this is too
+aggressive, you will quickly run into the daily rate limit.
+
+Since we cannot fetch up-to-date data very often, the binding will often reflect stale data. The
+Force Refresh action can be used to refresh on demand.
```

Comment:
```suggestion
Since we cannot fetch up-to-date data very often, the binding will often reflect stale data.
The Force Refresh action can be used to refresh on demand.
```